docs/source/dev_guide/review_process.rst

```diff
@@ -0,0 +1,76 @@
+.. _review_process:
+
+.. contents::
+   :local:
+
+Review process
+--------------
+
+This short description should serve as a rough guideline on how we deal with
+reviews on pull requests (PR). Some PRs are easy, others are hard. Some require
+multiple changes and continous feedback, others can be merged in flawlessly. So
+this text should communicate our intent and style and not be a go/no-go
+situation.
+
+After successfully adding a new feature, doing maintenance, sorting out a bug or
+adding new documentation, in short: contributing to the codebase in any form,
+the tests we have employed should all pass. These include tests for
+functionality on the C++ as well as the python side but also include checks for
+code style as far as they can be automated (see our :ref:`Conding convention
+<coding-convention>` for details on the style). We also have a custom target for
+make to help with autoformatting C++ as well as Python code.
+
+Further, we provide a PR template on github with several checkmarks to help
+start and streamline the process. Please make use of it and get rid of all
+unnecessary points, which just globber the PR message. To make it easy for
+possible reviewers do not hesitate to contact any of the people invovled in the
```

Comment:
Not sure I like the wording of these last two sentences. First, we want to emphasize that the PR checklist is the _main_ thing you can do to make it easier for potential reviewers. Second, we want to say that you should also feel free to contact the librascal developers about a PR if you're unsure about something.

But I think we should rather encourage PRs as a way of _starting_ the conversation about a new feature or change, and encourage draft PRs (as I wrote in the contribution guidelines!) rather than just opening a branch and letting it rot.

Finally, it's a bit unfair to mention things being "easier to talk about in person" when we might soon have collaborators in other parts of the world who have no possibility of talking to us in person (meaning: face to face). It's fine to say "don't hesitate to contact us", but I'd leave out the bit about conversations in person, since we all pretty much know that already.

Reply:
Oh, and "involved" (typo) if you decide to keep it